tools/biolatency.py

```diff
@@ -10,6 +10,7 @@
 # Licensed under the Apache License, Version 2.0 (the "License")
 #
 # 20-Sep-2015   Brendan Gregg   Created this.
+# 28-Jun-2017   Taekho Nam      Not stripped leading zero.
```

Comment:
I'm sorry, but there are no changes in biolatency and funclatency any more, so these comments also seem unnecessary.